cybersecurity/defensive/malware/virustotal.yml

```diff
@@ -0,0 +1,28 @@
+description: Analyse suspicious files, domains, IPs and URLs to detect malware and other breaches, automatically share them with the security community.
+
+functions:
+  virustotal_hash_lookup:
+    description: Lookup a given hash with VirusTotal.
+    parameters:
+      hash:
+        type: string
+        description: MD5/SHA1/SHA256 hash of file
+        examples:
+          - d41d8cd98f00b204e9800998ecf8427e  # MD5 example
+          - da39a3ee5e6b4b0d3255bfef95601890afd80709  # SHA1 example
+          - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  # SHA256 example
+      api_key:
+        type: string
+        description: Your VirusTotal API key
+        examples:
+          - 0123456789abcdef
+    container:
+      platform: linux/amd64
+      image: malice/virustotal
+    cmdline:  # https://hub.docker.com/r/malice/virustotal
+      - virustotal
+      - --api
+      - ${api_key}
```

Comment:
mmm i'm thinking about what's the best way to handle these cases ... maybe something like ${env.api_key} to read environment variables ... will add it to robopages-cli!

Reply:
sounds good, thanks! this would be awesome as opens a lot of opportunities for some great robopages. 
i'll await (no rush) to update this robopages then make some more env-related contributions. i was trying to tackle google/git dorking last night but seems not possible unless there is an API key so this would be cool

Reply:
there it is! https://github.com/dreadnode/robopages-cli/commit/ce5ea1ba6d0eab3e42776b1870425ed0ed489bf9

`${env.VIRUSTOTAL_API_KEY}`

`${env.SOMETHING_ELSE or default-value}`

Reply:
![image](https://github.com/user-attachments/assets/652879f9-15ec-4dd3-972f-aa0ffb2b1ab1)
🔥 🔥 🔥

Reply:
[chore: update to env!!](https://github.com/dreadnode/robopages/pull/9/commits/0556b89b6281aeb3ccd4ec2c936b44793233831c)

thank you so much!! lightning quick!! 🔥 hopefully that should be good i believe :)
i had another side idea from this great addition (non-urgent), wdyt if there was a per-robopage parameter/tag/label (whatever) for something like:

`requires_api_key: <boolean>`

Reply:
i think that api_key can be removed from the parameters now

Reply:
(if we leave it in the command line but remove it from the params, it'll be read from the environment variables and the model won't need to provide it)

Reply:
your absolutely right, apologies my oversight! thanks for the awesome catch